tests/unit_tests/build_test.py

```diff
@@ -0,0 +1,46 @@
+"""Unit tests for 'build' sub-command."""
+from diffkemp.cli import run_from_cli
+from diffkemp.utils import get_functions_from_llvm
+from tempfile import mkdtemp
+import os
+import pytest
+import shutil
+import yaml
+
+SINGLE_C_FILE = os.path.abspath("tests/testing_projects/make_based/file.c")
+MAKE_BASED_PROJECT_DIR = os.path.abspath("tests/testing_projects/make_based")
+
+
+@pytest.mark.parametrize("source",
+                         [SINGLE_C_FILE, MAKE_BASED_PROJECT_DIR],
+                         ids=["single c file", "make-based project"])
+def test_build_command(monkeypatch, source):
+    """Testing 'build' sub-command for single c file and make-based project."""
+    output_dir = mkdtemp()
+    # Running diffkemp build command
+    args = ["bin/diffkemp", "build", source, output_dir]
+    monkeypatch.setattr("sys.argv", args)
+    run_from_cli()
```

Comment:
Probably not (maximally in case if the names of arguments would change), but this approach seemed simpler to me than creating the argument object (the `run_from_cli` function creates it for us).